src/artemis/fast_grid_scan_plan.py

```diff
@@ -91,7 +102,11 @@ def get_plan(parameters: FullParameters):
     )
     zebra = Zebra(name="zebra", prefix=f"{parameters.beamline}-EA-ZEBRA-01:")
 
-    return run_gridscan(fast_grid_scan, zebra, eiger, parameters)
+    undulator = Undulator(
+        name="undulator", prefix=f"{parameters.beamline}-MO-SERVC-01:"
```

Comment:
**must**: This is still using the beamline prefix rather than the insertion prefix.

Reply:
Whoops. Fixed.